### DISCUSSION TITLE
IP Addresses blocked from Github Pages

### DISCUSSION THREAD

**Discussion by okram** [community/community#27813] [Other Feature Feedback, Questions, & Ideas]
Hello,

I live on an Indian reservation and use a small local Internet carrier that seems to have been blocked by GitHub Pages. We can no longer access project webpages.

```
~$ ping 185.199.110.153
PING 185.199.110.153 (185.199.110.153): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
```

However, when using VPN (e.g. and routing traffic through another country), we're able to access GitHub Pages. Unfortunately, not everyone in the tribe has VPN and we are wondering why our IPs have been blocked? For security reasons, I've included only the first 2 blocks of the IP4 range: `208.85.*.*`. Hopefully, that is sufficient to track down the blacklisted IPs.

If there has been some "abuse" from this IP range, can you please provide insight as to what the abuse was and also, why an entire IP block was banned from accessing GitHub pages (as I don't believe everyone in our small community has done something to "wrong" GitHub).

Thank you for your time.

**Comment by jenwatkins** [comment]
I too have a 208.85.*.* IP address and cannot reach GitHub Pages. I noticed the problem this morning. Thanks @okram for pointing this out. I hope Github will resolve the problem soon.

**Comment by yoannchaudet** [comment]
Hello,

I suggest reaching out to your local internet carrier. GitHub Pages has no traffic filters in place to block select carriers.

**Comment by jfullstackdev** [comment]
yup, I do agree, it maybe on your side, maybe you can ask your local provider

or maybe it's on your Settings, like the firewall or antivirus is blocking

**Reply by okram** [reply]
An "awk-to-the-sed-to-grep-to-the-pipe"-type colleague confirmed that it isn't GitHub pages. Apologies for taking your time. I have ordered my ragtag battalion of mercenaries to direct all fire at our local Internet provider ---- "chaaaaaaaarge!"
